maat-court-data-api/src/main/java/gov/uk/courtdata/model/assessment/PassportAssessment.java

```diff
@@ -0,0 +1,55 @@
+package gov.uk.courtdata.model.assessment;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+
+@Data
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+public class PassportAssessment {
+    private Integer repId;
+    private String nworCode;
+    private Integer cmuId;
+    private LocalDateTime assessmentDate;
+    private String partnerBenefitClaimed;
+    private String partnerFirstName;
+    private String partnerSurname;
+    private String partnerOtherNames;
+    private String partnerNiNumber;
+    private LocalDateTime partnerDob;
+    private String incomeSupport;
+    private String jobSeekers;
+    private String statePensionCredit;
+    private String under18FullEducation;
+    private String under16;
+    private String pcobConfirmation;
+    private String result;
+    private String dwpResult;
+    private String passportNote;
+    private String between16And17;
+    private String under18HeardInYouthCourt;
+    private String under18HeardInMagsCourt;
+    private LocalDateTime lastSignOnDate;
+    private String esa;
+    private String pastStatus;
+    @Builder.Default
+    private String replaced = "N";
+    private LocalDateTime passportEvidenceDueDate;
+    private LocalDateTime allPassportEvidenceReceivedDate;
+    private Integer passportUpliftPercentage;
+    private LocalDateTime passportUpliftApplyDate;
+    private LocalDateTime passportUpliftRemoveDate;
```

Comment:
The following fields all relate to passport assessment evidence, which isn't handled by the current implementation. I think for now these could probably be removed:

passportEvidenceDueDate
allPassportEvidenceReceivedDate
passportUpliftPercentage
passportUpliftApplyDate
passportUpliftRemoveDate
passportEvidenceNotes

Reply:
These will be needed once the passport evidence APIs are added in. Since we do not know how the business logic would be implemented in means assessment, it might be better to leave these in